from django.core.management.base import BaseCommand, CommandError
from core.settings import BASE_DIR
import pandas as pd
from Main.models import MD5MalwareHashes
class Command(BaseCommand):
    def handle(self, *args, **kwargs):
        filepath=str(BASE_DIR) + '/MalwareModel/MD5MalwareHashes.csv'    
        df = pd.read_csv(filepath)
        for row in df.iterrows():
            print(row["MD5"])
            try:
                MD5MalwareHashes.objects.create(MD5=row["MD5"])
                print(row["MD5"] + " added to the database")
            except Exception as e:
                print(e)